packages/openapi-v3/src/spec-builder/openapi-v3-spec-builder.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// If we agree to merge `@loopback/openapi-spec-builder` and `@loopback/openapi-v3`,
```

Comment:
What's the reasoning behind merging those two modules? I think it's okay to keep them separate so that `openapi-spec-builder` can be used independently.

Reply:
@b-admike Good question, it reminds me the discussion why we don't merge them!
`@loopback/openapi-spec-builder` is used for top-down build of OpenAPI spec
`@loopback/openapi-v3` is used for bottom-up build.